pep-0001.txt

```diff
@@ -281,14 +281,24 @@ notifying the PEP author(s) and giving them a chance to make revisions.
 The final authority for PEP approval is the Steering Council. However, whenever
 a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
-the PEP-Delegate for that PEP, and they will then have the authority to approve
-(or reject) that PEP. Individuals taking on this responsibility are free to seek
+the PEP-Delegate (BDFL-Delegate) for that PEP, and they will then have the
+authority to approve (or reject) that PEP. 
+
+The terms, PEP-Delegate (preferred term under the Steering Council governance 
+model) and BDFL-Delegate, may be used interchangeably in discussion.
+The term "PEP-Delegate" will be used under the Steering
+Council governance model. The PEP's designated decision maker,
```

Comment:
The second sentence is a stronger phrasing than the first ("will be used" vs "preferred term"). I'd suggest keeing the latter, and moving the explanation that the terms mean the same thing to the end of the paragraph.